packages/expo/Expo.podspec

```diff
@@ -97,6 +100,8 @@ Pod::Spec.new do |s|
   s.dependency 'React-RCTFabric'
   s.dependency 'ReactAppDependencyProvider'
 
+  install_modules_dependencies(s)
```

Comment:
given that this is added, it's possible this podspec could be further simplified https://github.com/facebook/react-native/blob/4b82896390592f2b1678ac96aaa4b35cbb97f64c/packages/react-native/scripts/cocoapods/new_architecture.rb#L73

can be done later though

Reply:
yep it's actually added to minimize newly deps or header search paths in Expo.podspec.
for existing code, i would leave it for the other round to clean up. thanks!